x-pack/plugins/apm/server/routes/metrics/route.ts

```diff
@@ -105,7 +109,64 @@ const serviceMetricsJvm = createApmServerRoute({
   },
 });
 
+const serverlessMetricsRoute = createApmServerRoute({
+  endpoint: 'GET /internal/apm/services/{serviceName}/metrics/serverless',
+  params: t.type({
+    path: t.type({
+      serviceName: t.string,
+    }),
+    query: t.intersection([environmentRt, kueryRt, rangeRt]),
+  }),
+  options: { tags: ['access:apm'] },
+  handler: async (
+    resources
+  ): Promise<{
+    metricCharts: Awaited<ReturnType<typeof getServerlessAgentMetricsCharts>>;
+    serverlessSummary: Awaited<ReturnType<typeof getServerlessSummary>>;
+    serverlessFunctionsOverview: Awaited<
+      ReturnType<typeof getServerlessFunctionsOverview>
+    >;
+    serverlessActiveInstancesOverview: Awaited<
+      ReturnType<typeof getServerlessActiveInstancesOverview>
+    >;
+  }> => {
+    const { params } = resources;
+    const setup = await setupRequest(resources);
+
+    const { serviceName } = params.path;
+    const { environment, kuery, start, end } = params.query;
+    const options = {
+      environment,
+      start,
+      end,
+      kuery,
+      setup,
+      serviceName,
+    };
+
+    const [
+      metricCharts,
+      serverlessSummary,
+      serverlessFunctionsOverview,
+      serverlessActiveInstancesOverview,
+    ] = await Promise.all([
```

Comment:
I think it makes sense to split some of these in separate requests for incremental loading like other pages e.g. (service overview).